test/kitchen/test-definitions/chef-test.yml

```diff
@@ -13,7 +13,7 @@ suites:
         enable: false
     datadog:
       <% dd_agent_config.each do |key, value| %>
-      <%= key %>: <%= value %>
+      <%= key %>: "<%= value %>"
```

Comment:
Do we need to update this definition here since we're overriding it in `process-agent-test.yml`?

Reply:
Yes, we need to update all definitions to escape the values, since now one of them (the `aptrepo` which now has brackets) would break the yaml file if not escaped.

Reply:
Thanks for explaining it, Kylian!